README.md

```diff
@@ -11,56 +17,21 @@ All templates from this repository come as part of the [Amido.Stacks.CQRS.Templa
 
 #### Template installation
 
-To install the templates on your own machine you'll need to download the [Amido.Stacks.CQRS.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/) NuGet package.
-
-Then you can install it locally via the command line (command is given with an example version)
+For the latest template version, please consult the Nuget page [Amido.Stacks.CQRS.Events.Templates](https://www.nuget.org/packages/Amido.Stacks.CQRS.Templates/). To install the templates to your machine via the command line:
```

Comment:
Name should be `Amido.Stacks.CQRS`. The link is correct.

Reply:
Link changed to Amido.Stacks.CQRS